tests/MongoDB.Analyzer.Tests/Infrastructure/DriverVersionHelper.cs

```diff
@@ -12,21 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System;
 using System.Linq;
 using NuGet.Versioning;
 
 namespace MongoDB.Analyzer.Tests.Infrastructure;
 
 internal static class DriverVersionHelper
 {
-    public static readonly NuGetVersion[] DriverVersions = new[]
+    public static readonly NuGetVersion[] DriverVersions;
+
+    static DriverVersionHelper()
+    {
+        var driverVersion = Environment.GetEnvironmentVariable("DRIVER_VERSION");
+
+        if (driverVersion != null)
```

Comment:
IMHO: it would be better just to expect of Environment variable to be set. Because otherwise it could hide missed variable, by using defaults.

Reply:
We need those presets for testing in IDE.
But good point, maybe we can think of some other trick.

Reply:
In Rider there is a way to specify Environment vars for unit tests. I suppose VS could have something similar.